docs/content/docs/recipes/teams.md

```diff
@@ -207,8 +207,8 @@ const getFluidContainer = async (url : URLSearchParams) => {
 };
 ```
 
-1. Replace `TODO 5` with the following code. Now that we've defined how to get our Fluid container, you need to tell React to call `getFluidContainer` on load, and then store the result in state based on if we are inside Teams.
-React's [useState hook](https://reactjs.org/docs/hooks-state.html) will provide the storage needed, and [useEffect](https://reactjs.org/docs/hooks-effect.html) will allow us to call `getFluidContainer` on render, passing the returned value into `setFluidContainer`.
+1. Replace `TODO 5` with the following code. Now that you've defined how to get the Fluid container, you need to tell React to call `getFluidContainer` on load, and then store the result in state based on if the application is inside Teams.
+React's [useState hook](https://reactjs.org/docs/hooks-state.html) will provide the storage needed, and [useEffect](https://reactjs.org/docs/hooks-effect.html) will allow you to call `getFluidContainer` on render, passing the returned value into `setFluidContainer`.
```

Comment:
This results in storing data "twice" right? Once in Fluid, and once in React state?

Reply:
so I believe React `setState` is a shallow copy, so it'll be copied by reference. However, even if it's not, the `const container = await getContainer(containerId);` would only be scoped within the `getFluidContainer` function and would be disposed after the function scope ends. So it's important that we set the value into the state so the memory location and contents are preserved.